index.bs

```diff
@@ -445,9 +451,12 @@ run these steps:
 
 1. Let |accepts options| be a empty [=/list=] of [=tuples=]
    consisting of a description and a filter.
-1. [=list/For each=] |type| of |options|.{{FilePickerOptions/types}}:
-  1. Let |description| be |type|.{{FilePickerAcceptType/description}}.
-  1. [=map/For each=] |typeString| → |suffixes| of |type|.{{FilePickerAcceptType/accept}}:
+1. [=list/For each=] |type| of |options|["{{FilePickerOptions/types}}"]:
+  1. Let |description| be |type|["{{FilePickerAcceptType/description}}"] if
```

Comment:
Not this PR, but it seems like we can make this "" (empty string) by default in the IDL, then we don't need this in prose?

Reply:
Good catch. I'll change this in a follow-up